vscode-extensions/tls-sync-vscode-shared/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@rushstack/tls-sync-vscode-shared",
```

Comment:
Generalize this package to be a VSCode shared utilities package, not just for TLS sync